source/ControllableHero.hx

```diff
@@ -0,0 +1,33 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+
+class ControllableHero extends FlxSprite
+{
+	public function new(x:Float = 0, y:Float = 0)
+	{
+		super(x, y);
+
+		makeGraphic(20, 20, 0xFFD4AF37);
+	}
+
+	override function update(elapsed:Float)
+	{
+		if (FlxG.keys.pressed.UP)
+			velocity.y = -50;
+		else if (FlxG.keys.pressed.DOWN)
+			velocity.y = 50;
+		else if (FlxG.keys.pressed.LEFT)
+			velocity.x = -50;
+		else if (FlxG.keys.pressed.RIGHT)
+			velocity.x = 50;
+		else
+		{
+			velocity.x = 0;
+			velocity.y = 0;
+		}
```

Comment:
There is a little wonkiness in movement that comes from locking the "reset velocity to zero" logic behind this else statement - to demonstrate, try holding left or right, then pressing up or down while continuing to hold to the side.

Not essential to fix, but it seems like a good puzzle to figure out if you've got the time.

Reply:
I think I can adapt something I saw in a tutorial for this using drag and cancelling out multiple key presses.  I'll come back to this.